world_universities_and_domains.json

```diff
@@ -47873,13 +47876,14 @@
   },
   {
     "web_pages": [
-      "http://www.da-iict.org/"
+      "https://www.daiict.ac.in/"
     ],
     "name": "Dhirubhai Ambani Institute of Information and Communication Technology",
     "alpha_two_code": "IN",
     "state-province": null,
     "domains": [
-      "da-iict.org"
+      "da-iict.org",
```

Comment:
The domain might be obsolete already because I cannot access http://www.da-iict.org/ anymore. However, would be better to keep it there for "backward compatibility" 👍